src/components/DataTable/DataTable.scss

```diff
@@ -161,11 +165,16 @@ $breakpoint: 768px;
 .Cell-total-footer {
   border-top: border('divider');
   border-bottom: none;
+  border-bottom-left-radius: var(--p-border-radius-base, border-radius());
+  border-bottom-right-radius: var(--p-border-radius-base, border-radius());
```

Comment:
Do we still need the border-raidus on the bottom when the border-top is used?

Reply:
I believe so, this is unrelated to the border-top and is added because footers are the only part missing the curved edge.

Reply:
shipit